esp_jpeg/test/tjpgd_test.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <string.h>
```

Comment:
Please add license + author header

Reply:
No change in esp_jpeg component. Only added test and moved from BSP. I increased the version for upload changed URL to component manager.